packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -1389,7 +1396,6 @@ describe('ReactDOMInput', () => {
       'node.value = "1980-01-01"',
       'node.setAttribute("value", "1980-01-01")',
       'node.setAttribute("checked", "")',
-      'node.setAttribute("checked", "")',
```

Comment:
Just so I can better understand this change, is the reduction of this setAttribute call due to the following case?

**`checked` or `defaultChecked` are `true`**

```javascript
// <input defaultChecked=true} />

node.defaultChecked = true // node.defaultChecked = !node.defaultChecked
node.defaultChecked = true // node.defaultChecked = !!true
```

Reply:
Sorry. Didn't catch your comment earlier. This makes sense!